src/rime/algo/syllabifier.cc

```diff
@@ -121,6 +155,10 @@ int Syllabifier::BuildSyllableGraph(const string &input,
       // when there is a path of more favored type
       SpellingType edge_type = kInvalidSpelling;
       for (auto k = j->second.begin(); k != j->second.end(); ) {
+        if (k->second.is_correction) {
+          ++k;
+          continue; // Don't care correction edges
```

Comment:
Again, what if only correction can form a path to the vertex?

Reply:
Corrections should not take any effect in post processing phrase.
My purpose is to keep the correction part separated from the origin syllable graph.